R/commonMachineLearningClassification.R

```diff
@@ -15,254 +15,236 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-.readDataClassificationAnalyses <- function(dataset, options) {
-  if (is.null(dataset))
+.mlClassificationReadData <- function(dataset, options) {
+  if (is.null(dataset)) {
     dataset <- .readDataClassificationRegressionAnalyses(dataset, options)
-
-  if (length(unlist(options[["predictors"]])) > 0 && options[["scaleEqualSD"]])
-    dataset[,options[["predictors"]]] <- .scaleNumericData(dataset[,options[["predictors"]], drop = FALSE])
-
-  if (options[["target"]] != "")
+  }
+  if (length(unlist(options[["predictors"]])) > 0 && options[["scaleEqualSD"]]) {
+    dataset[, options[["predictors"]]] <- .scaleNumericData(dataset[, options[["predictors"]], drop = FALSE])
+  }
+  if (options[["target"]] != "") {
     dataset[, options[["target"]]] <- factor(dataset[, options[["target"]]])
-
+  }
   return(dataset)
 }
 
-.errorHandlingClassificationAnalyses <- function(dataset, options, type) {
+.mlClassificationErrorHandling <- function(dataset, options, type) {
   .errorHandlingClassificationRegressionAnalyses(dataset, options, type)
-
   # add checks specifically for classification analyses here
 }
 
-.classificationAnalysesReady <- function(options, type){
-  if(type == "lda" || type == "randomForest" || type == "boosting"){
+.mlClassificationReady <- function(options, type) {
+  if (type == "lda" || type == "randomForest" || type == "boosting") {
     ready <- length(options[["predictors"]][options[["predictors"]] != ""]) >= 2 && options[["target"]] != ""
   } else if (type == "knn" || type == "neuralnet") {
     ready <- length(options[["predictors"]][options[["predictors"]] != ""]) >= 1 && options[["target"]] != ""
   }
   return(ready)
 }
 
-.classificationFormula <- function(options, jaspResults){
+.classificationFormula <- function(options, jaspResults) {
```

Comment:
Perhaps call this function `.setClassificationFormula` or `.classificationSetFormula`. In general, functions should be verbs, as they do something, whereas variables should be nouns, as they are something. What caught my eye was when this function was called, and I didn't know what it was doing. The "set" part tells me that a property of an object is set, where the object is jaspResults. Not sure if this is reasonable perhaps @vandenman has something to say about this.